.python-version

```diff
@@ -0,0 +1 @@
+3.6.8
```

Comment:
Could you add a comment explaining that this is tied to the Python version in RHEL 8?

Reply:
It looks like comments are not allowed.